src/main/java/knightminer/inspirations/tools/datagen/ToolsRecipeProvider.java

```diff
@@ -96,7 +94,6 @@ protected void registerRecipes(@Nonnull Consumer<IFinishedRecipe> consumer) {
 		CondRecipe.shaped(InspirationsTools.key)
 				.addCondition(TOOLS)
 				.addCondition(ConfigEnabledCondition.LOCK)
-				.canMirror()
```

Comment:
Why remove canMirror from here? Does it default to true? If not, this recipe certainly should support mirroring

Reply:
As far as I can tell that option doesn't exist anymore, it seems to always be applied.

Reply:
Nevermind, saw your commit message.